Assets/Mirror/Components/InterestManagement/SpatialHashing/SpatialHashingInterestManagement.cs

```diff
@@ -100,16 +99,23 @@ internal void Update()
 
                     // put into grid
                     grid.Add(position, connection);
+
+                    if (projectNearClientOwnedObjects)
+                        foreach (NetworkIdentity clientOwnedObject in connection.clientOwnedObjects)
+                        {
+                            Vector2Int clientOwnedObjectPosition = ProjectToGrid(clientOwnedObject.transform.position);
+                            grid.Add(clientOwnedObjectPosition, connection);
+                        }
                 }
             }
 
             // rebuild all spawned entities' observers every 'interval'
             // this will call OnRebuildObservers which then returns the
             // observers at grid[position] for each entity.
-            if (NetworkTime.localTime >= lastRebuildTime + rebuildInterval)
+            if (NetworkTime.time >= lastRebuildTime + rebuildInterval)
             {
                 RebuildAll();
-                lastRebuildTime = NetworkTime.localTime;
+                lastRebuildTime = NetworkTime.time;
```

Comment:
Merge master into this PR to eliminate these changes

Reply:
Done

Reply:
Now it's showing a complete file replacement - did you change indention or tabs instead of spaces?

Reply:
Now it should be ok